bitbots_utils/launch/tools.launch

```diff
@@ -1,8 +1,19 @@
 <?xml version="1.0"?>
 <launch>
-    <include file="$(find-pkg-share system_monitor)/launch/system_monitor.launch">
-        <arg name="required" value="false" />
-    </include>
+    <arg name="monitoring" default="true" description="true: start system monitor and udp bridge" />
```

Comment:
I am still not sure if it fits into the monitoring. E.g. the camera launch file is also not in the monitoring. 

For the rosbag record, okay that's fine.